client/accounts_admin.html

```diff
@@ -42,7 +42,11 @@
     {{/each}}
     </tbody>
   </table>
-  <div class="form-inline">
+  <div class="text-center">  
+    {{> defaultBootstrapPaginator pagination=templatePagination limit=10 containerClass=""}}
```

Comment:
Add class `text-center` as value of attribute `containerClass` and remove upper `div` block. 
It allows the same design that you made but using a tool of pagination button